src/sentry/workflow_engine/migration_helpers/issue_alert_migration.py

```diff
@@ -161,7 +182,16 @@ def _create_workflow_and_lookup(
         detector: Detector,
     ) -> Workflow:
         when_dcg = self._create_when_dcg(action_match=action_match)
-        self._bulk_create_data_conditions(conditions=conditions, filters=filters, dcg=when_dcg)
+        data_conditions = self._bulk_create_data_conditions(
+            conditions=conditions, filters=filters, dcg=when_dcg
+        )
+
+        # the only time the data_conditions list will be empty is if somebody only has EveryEventCondition in their conditions list.
+        # if it's empty and this is not the case, we should not migrate
+        if len(data_conditions) == 0 and not (
+            len(conditions) == 1 and conditions[0]["id"] == EveryEventCondition.id
```

Comment:
```suggestion
        no_conditions = len(data_conditions) == 0
        only_has_every_event_cond = len(conditions) == 1 and conditions[0]["id"] == EveryEventCondition.id
        
        if no_conditions and not only_has_every_event_cond:
```

basically, i just find it a little easier to read the `if` statement if we give the variables names.